Source/Scene/BillboardCollection.js

```diff
@@ -1465,7 +1465,8 @@ define([
                 this._rsTranslucent = RenderState.fromCache({
                     depthTest : {
                         enabled : true,
-                        func : WebGLConstants.LEQUAL  // Allows label glyphs and billboards to overlap.
+                        // Allows label glyphs and billboards to overlap.
+                        func : ((this._blendOption === BlendOption.TRANSLUCENT) ? WebGLConstants.LEQUAL : WebGLConstants.LESS)
```

Comment:
Good eye, but no.  `this._blendOption === BlendOption.TRANSLUCENT` means that the entire billboard is being written in the translucent pass, without an opaque pass.  You'll notice on [#L1471](https://github.com/AnalyticalGraphicsInc/cesium/blob/e30f96dd41f3ada251f9f660fab00dd078611405/Source/Scene/BillboardCollection.js#L1471) that we already use the same test to determine whether or not to write to the depth buffer.  This is because, when the opaque pass is NOT in use, we rely on the depth buffer to save us from OIT making everything blend with everything, even if it means "punching holes" in things.  When the 2-pass system is in use, we're avoiding the "punching holes" bug by using the depth buffer only in the opaque pass and not the translucent pass.

So, the difference between single-pass and 2-pass controls a difference in strategy of how we use the depth buffer, and thus should also control the depth test function.  In single-pass mode we use `LEQUAL` because the depth buffer is in use and we need to overlap fragments at the same depth.  In 2-pass mode we use `LESS` to avoid the fringe overlap problem.

Basically, with this change, single-pass mode has returned to the behavior from before the "punching holes" fix, making it fully backwards-compatible.